code/modules/paperwork/photography.dm

```diff
@@ -425,9 +472,11 @@
 		viewpichelper(Ainfo)
 
 /obj/item/device/camera/afterattack(atom/target, mob/user, flag)
-	if(!on || !pictures_left || !isturf(target.loc))
+	if(!on || !pictures_left)
+		return
+	var/turf/T = locate(target.x,target.y,target.z)

```

Comment:
It wasn't taking pictures unless you aimed at something that was ON a turf instead of the turf itself, as a turf's loc is an area and not a turf so it didn't work. This makes sure it takes a picture of the turf.
....
Or I can just use get_turf() but uh, this doesn't need an if/else to check if it's already a turf.